src/main.cpp

```diff
@@ -1,9 +1,133 @@
-#include <cstdlib>
+#include <algorithm>
+#include <iostream>
+#include <iterator>
 #include <iostream>
+#include <cstdlib>
+#include <fstream>
+#include <sstream>
+#include <iomanip>
+#include <string>
+#include <vector>
+using namespace std;
+
+// Global Variables & Functions
+vector<string> all_types = {"PROGRAM", "VAR", "END_VAR", "END_PROGRAM", "BOOL", "TRUE", "FALSE"}; //keywords
+vector<string> operators = {"<", ">", "+", "-", "%", ":="}; // operators
+vector<string> all_sc = {"{", "}", "[", "]", ";", ":"}; // special characters
+void tokenizerFunc (string sample_input); // main lexer function
+typedef struct Token{ // where/how the keywords are going to be saved
```

Comment:
`typedef struct` is a holdover from C. In C++ structs are implicitly typedef-ed.

```suggestion
// where/how the keywords are going to be saved
struct Token{ 
```

Reply:
This token class should be in it's own file `token.hpp` and possibly `token.cpp` if it has any methods.

Reply:
I've created token header and cpp files.  I also removed "typedef" before struct.